test/test_automl/test_estimators.py

```diff
@@ -275,6 +289,27 @@ def test_cv_results(tmp_dir, output_dir):
                        cv_results.items() if key.startswith('param_')]
     assert all(cv_result_items), cv_results.items()
 
+    # Compare the state of the model parameters with the original parameters
+    new_params = clone(cls).get_params()
```

Comment:
Is there a specific reason you added this to the unit test for cv_results?

Reply:
I really just wanted to test an implementation fo autosklearnclassifier, nothing more. This can be done anywhere else and the reason for that is that clone of the estimator makes sure that the set/get parameters work. Do you think it should be somewhere else?

Reply:
No, it's totally fine to have it here.